src/components/SharingButtons.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import BaseComponent from 'lib/BaseComponent';
+
+export default class SharingButtons extends BaseComponent {
+
+  render() {
+    let renderButtons = () => {
+      let data = [];
```

Comment:
Yes

On Sun, Aug 7, 2016, 4:28 AM Zane Mountcastle <notifications@github.com>
wrote:

> In src/components/SharingButtons.jsx
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41#discussion_r73794775>
> :
>
> > @@ -0,0 +1,102 @@
> > +import React from 'react';
> > +import BaseComponent from 'lib/BaseComponent';
> > +
> > +export default class SharingButtons extends BaseComponent {
> > +
> > +  render() {
> > +    let renderButtons = () => {
> > +      let data = [];
>
> Sure I'll change that. I've used data as a variable for render methods in
> other classes. Should I change those to let nodes = [] as well?
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41/files/a85a19c83b836058745a11879e25ef14a8a7a67e#r73794775>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAFzb5fJdpKT4gS6pV_2AJwVwmeetuHks5qdULLgaJpZM4JdXZj>
> .
>